### ISSUE TITLE
Way to specify different style for selected item?

### ISSUE THREAD

**Issue by eligt** [yangyxd/flutter_picker#17]
Is there a way to have the selected item be instantiated separately or at least pass in a TextStyle to be used exclusively for the currently selected item?

**Comment by yangyxd** [comment]
TextStyle in Picker can be used.

**Comment by eligt** [comment]
Yes but I was requesting the possibility of passing an extra TextStyle that is exclusively used for the selected item, I don't think this is currently possible?

**Comment by yangyxd** [comment]
Yes, this is not supported.
